### ISSUE TITLE
How to add watermarker use chinese?

### ISSUE THREAD

**Issue by legiorange** [LibrePDF/OpenPDF#293]
I put OpenPDF in project 
but still don't know how to add watermarker use chinese.
I try use unicode but not chinese here . I try change character set now.
I really need help.

**Comment by noavarice** [comment]
Can you please provide us with minimal working example? This will simplify debug and testing

**Comment by legiorange** [comment]
1.package com.lowagie.toolbox.plugins.watermarker;
2.class WatermarkerTest 


    @Test
    void shouldWriteWatermarkWithIOStreams() throws IOException, DocumentException {
        // GIVEN
        Path path = Paths.get("src/test/resources/MyFile.pdf");
        byte[] input = readAllBytes(path);
        String text = "1 :\u4e2d\u6587";
        int fontsize = 100;
        float opacity = 0.5f;

        // WHEN
        byte[] result = new Watermarker(input, text, fontsize, opacity)
                .withColor(RED)
                .write();

        // THEN
        assertTrue(result.length > input.length);

        // Uncomment the following to write the generated file on disk
         writeOnDisk(result);
    }

---------------------------------------------

I run the test
 String text = "1 :\u4e2d\u6587";

 unicode to chinese is: 
 \u4e2d\u6587="中文"

  I  think things going right and open Result.pdf 
 
The watermaker like... "1:        "

**Comment by noavarice** [comment]
@legiorange, sorry for a such a long response
**TL;DR**: Currently it's not possible to add watermark with Chinese symbols, but you can workaround it for now.

**Explanation**: After some investigation it finds out that you cannot add watermark with Chinese text at least with current version of OpenPDF. The root of a problem is a font that is used to add watermark - it's Helvetica with CP-1252 single-byte encoding and AFAIU it cannot be changed from the `com.lowagie.toolbox.plugins.watermarker.Watermarker` API.

**Possible workaround**: you can create your own copy of `com.lowagie.toolbox.plugins.watermarker.Writer` and `com.lowagie.toolbox.plugins.watermarker.Watermarker`
 and change first line in `Writer#write`:
`BaseFont bf = createFont("/<path-to-file>/hxb-meixinti.ttf", IDENTITY_H, EMBEDDED); (Mei Xin Ti is just first CJK font I managed to find)`
Picture below is the result I got after tweaking `Writer#write`.
![image](https://user-images.githubusercontent.com/23114811/68997768-fd9bb200-08ba-11ea-8c1e-d6ba1843095f.png)

We will fix customization of watermark font in the future release since it should be easy, but if there are circumstances that you need this functionality right now - you can apply mentioned workaround.
Hope this helps

**Comment by legiorange** [comment]
I Love u。

It's helpful!

**Comment by asturio** [comment]
Closing, it works now :-)
